src/LanguageServer/Test/ImportsTests.cs

```diff
@@ -830,5 +832,39 @@ import module2
             comps = cs.GetCompletions(analysis, new SourceLocation(4, 9));
             comps.Should().HaveLabels("Y");
         }
+
+        [TestMethod, Priority(0)]
+        public async Task ZipFiles() {
+            var root = Path.Combine(GetAnalysisTestDataFilesPath(), "ZipEgg");
+            await CreateServicesAsync(root, PythonVersions.LatestAvailable3X, searchPaths: new[] { root, Path.Combine(root, "zipped.zip") });
+            var rdt = Services.GetService<IRunningDocumentTable>();
+            var analyzer = Services.GetService<IPythonAnalyzer>();
+
+            var uriPath = Path.Combine(root, "zip_user.py");
+            var code = await File.ReadAllTextAsync(uriPath);
+            var moduleUri = TestData.GetTestSpecificUri(uriPath);
+            var module = rdt.OpenDocument(moduleUri, code);
+
+            await analyzer.WaitForCompleteAnalysisAsync();
+            var analysis = await module.GetAnalysisAsync(-1);
+            analysis.Should().HaveVariable("i").OfType(BuiltinTypeId.Int);
+        }
+
+        [TestMethod, Priority(0)]
+        public async Task EggFiles() {
+            var root = Path.Combine(GetAnalysisTestDataFilesPath(), "ZipEgg");
+            await CreateServicesAsync(root, PythonVersions.LatestAvailable3X, searchPaths: new[] { root, Path.Combine(root, "test.egg") });
+            var rdt = Services.GetService<IRunningDocumentTable>();
+            var analyzer = Services.GetService<IPythonAnalyzer>();
+
+            var uriPath = Path.Combine(root, "egg_user.py");
+            var code = await File.ReadAllTextAsync(uriPath);
+            var moduleUri = TestData.GetTestSpecificUri(uriPath);
+            var module = rdt.OpenDocument(moduleUri, code);
+
+            await analyzer.WaitForCompleteAnalysisAsync();
+            var analysis = await module.GetAnalysisAsync(-1);
+            analysis.Should().HaveVariable("i").OfType(BuiltinTypeId.Int);
+        }
```

Comment:
So these are for adding the .zip and .egg files on the user search paths, I'm not exactly sure how to test for interpreter search paths. @AlexanderSher could you give me some suggestions?